script/color-contrast.ts

```diff
@@ -30,56 +30,62 @@ const getOpaqueColor = (color: string, background: string): string => {
   // parse BG
   const [bgR, bgG, bgB, bgAlpha] = parseToRgba(background)
   // return mixed color
-  const mixed = normal({ r: bgR, g: bgG, b: bgB, a: bgAlpha }, { r: colorR, g: colorG, b: colorB, a: colorAlpha })
+  const mixed = normal({r: bgR, g: bgG, b: bgB, a: bgAlpha}, {r: colorR, g: colorG, b: colorB, a: colorAlpha})
   return rgba(mixed.r, mixed.g, mixed.b, mixed.a)
 }
 /**
  * runContrastTest
  * @description runs through all color pairs of a theme and checks the contrasts
- * @param colorPairs 
- * @param colors 
+ * @param colorPairs
+ * @param colors
  * @returns contrastTestResult
  */
-const runContrastTest = (colorPairs: ContrastRequirement[], colors: any): contrastTestResult[] =>
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const runContrastTest = (colorPairs: ContrastRequirement[], scopedColors: any): contrastTestResult[] =>
   // Object.fromEntries(
   colorPairs.flatMap(([minimumContrast, colorA, colorB, options]: ContrastRequirement) => {
     // concat name
     const contrastPair = `${colorA} vs. ${colorB}`
     // build required string
-    const minimumContrastRatio = `${minimumContrast}:1`;
+    const minimumContrastRatio = `${minimumContrast}:1`
     // colorB is fully opaque
-    if (parseToRgba(colors[colorB])[3] === 1) {
+    if (parseToRgba(scopedColors[colorB])[3] === 1) {
       return {
         contrastPair,
-        ...testContrast(minimumContrast, colors[colorA], colors[colorB], undefined, contrastPair),
+        ...testContrast(minimumContrast, scopedColors[colorA], scopedColors[colorB], undefined, contrastPair),
         minimumContrastRatio
       }
     }
-    // if colorB is semi-transparent 
-    // get the correct canvas colors to test agains
-    let canvasColorArrays = cavnasColors
+    // if colorB is semi-transparent
+    // get the correct canvas colors to test again
```

Comment:
```suggestion
    // get the correct canvas colors to test against
```

Reply:
ah right.. was trying to fix a typo and ended up creating another one 😅